test/jdk/javax/swing/JTabbedPane/4209065/bug4209065.java

```diff
@@ -49,9 +50,14 @@ private void createTabbedPane() {
         JTabbedPane tp = new JTabbedPane();
         getContentPane().add(tp);
         String text = "<html><center>If the style of the text on the tabs matches"
-                      + "<br>the descriptions, press <em><b>PASS</b></em></center></html>";
-        tp.addTab("<html><center><font size=+3>big</font></center></html>", new JLabel(text));
-        tp.addTab("<html><center><font color=red>red</font></center></html>", new JLabel(text));
-        tp.addTab("<html><center><em><b>Bold Italic!</b></em></center></html>", new JLabel(text));
+                      + "<br>the descriptions, press <em><b>PASS</b></em><br>"
+                      + "<br> NOTE: the text (tab title) may not always "
+                      + "fit the tab.</center></html>";
```

Comment:
How about 
NOTE: where a large font is used, the text may be larger than the tab height but this is OK and not a failure.

Reply:
@prrace Sounds more clear. I'm currently working on changing the test case to use PassFailFrame as well and will include this change along with the other changes.

Reply:
@prrace @prsadhuk This PR might require a re-review since the test case has new changes.